packages/traceability/issuer-api/src/pods/certificate/dto/certificate.dto.ts

```diff
@@ -68,7 +68,7 @@ export class CertificateDTO {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     blockchain?: any;
 
-    @ApiProperty({ type: String })
+    @ApiProperty({ type: String, example: "x8888"})
```

Comment:
Transaction hashes are usually in a different format:
[0x2b8da531e46cff1e217abc113495befac9384339feb10816b0f7f2ffa02fadd4](https://explorer.energyweb.org/tx/0x2b8da531e46cff1e217abc113495befac9384339feb10816b0f7f2ffa02fadd4)